### DISCUSSION TITLE
All classes probabilities for particular image

### DISCUSSION THREAD

**Discussion by Venky0892** [ultralytics/yolov5#6939] [General]
Hi, is it possible to get all the class probabilities in the output of an image just to ensure how good the model is capable of detecting an object ?

**Comment by glenn-jocher** [comment]
@Venky0892 no. This information is only available at the NMS input:
https://github.com/ultralytics/yolov5/blob/055e72af5b887832d5e7267ac9226c825d498cd2/utils/general.py#L699-L706
